server/container/guice/guice-common/src/main/java/org/apache/james/GuiceJamesServer.java

```diff
@@ -75,10 +74,6 @@ public void stop() {
         }
     }
 
-    public GuiceServerProbe serverProbe() {
```

Comment:
Are you supposed to guess the name of the probe you want to get?
Isn't it possible to add some helpers for each available probe?